include/cute/atom/copy_traits_xe.hpp

```diff
@@ -2222,21 +2316,22 @@ struct Copy_Traits<COPY_OP, args_t...> : Copy_Traits_<COPY_OP, args_t...>{ \
       : Copy_Traits_<CopyOp, args_t...>(args...) {} \
 };
 
-COPY_TRAIT_LD_DEF(XE_2D_U8x1x32_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x2x32_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x4x32_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x8x32_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x1x64_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x2x64_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x4x64_LD_N)
-COPY_TRAIT_LD_DEF(XE_2D_U8x8x64_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x1x32_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x2x32_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x4x32_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x8x32_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x1x64_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x2x64_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x4x64_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x8x64_LD_N)
 COPY_TRAIT_LD_DEF(XE_2D_U64x8x1_LD_T)
 COPY_TRAIT_LD_DEF(XE_2D_U64x8x2_LD_T)
 COPY_TRAIT_LD_DEF(XE_2D_U64x8x4_LD_T)
-COPY_TRAIT_LD_DEF(XE_2D_U8x16x32_LD_N)
+COPY_TRAIT_LD_DEF(XE_2D_Packed_U8x16x32_LD_N)
```

Comment:
Hi, what's the connotation of `Packed`?

It seems `XE_2D_Packed_U8x16x32_LD_N` is not appropriate for loading `A` matrix elements for FP8 GEMM.

Reply:
Resolved. Read it in your documentation PR. Thanks!